services/dag-indexer/src/mappings/handleCall.ts

```diff
@@ -0,0 +1,77 @@
+global.TextEncoder = require('util').TextEncoder
+global.TextDecoder = require('util').TextDecoder
+global.Buffer = require('buffer/').Buffer
```

Comment:
These are duplicated in [polyfill-text-encoding.ts](https://github.com/autonomys/auto-files-gateway/pull/131/files#diff-f1f6e16ea4d56845269b7edf65751eca7113d9a0a03c0786d73eca1a31192224)

Reply:
Refactored into [polyfill-text-encoding.ts](https://github.com/autonomys/auto-files-gateway/pull/131/files#diff-f1f6e16ea4d56845269b7edf65751eca7113d9a0a03c0786d73eca1a31192224)